futaba/cogs/moderation/core.py

```diff
@@ -201,6 +204,15 @@ async def jail(self, ctx, member: MemberConv, *, reason: str = None):
 
         await self.perform_jail(ctx, member, 0, reason)
 
+    async def on_message(self, message):
+        """
+        Bot Event.
+        :param message: Messages.
+        :return: Nothing.
+        """
+        if len(message.mentions) > 5:
+           await self.perform_jail_internal(message, message.author, 0, "Mention Bombing.")
```

Comment:
is there any reason this needs to take `message.author` separately?

Reply:
I thought that it would break ``perform_jail`` where it passes in a specific member into it if I did it that way. I probably could have it where if ``None`` is passed in that 2nd param that it would default me message.author though if needed.